apps/provisioning_api/lib/Controller/AUserData.php

```diff
@@ -137,6 +137,9 @@ protected function getUserData(string $userId, bool $includeScopes = false): ?ar
 		$groups = $this->groupManager->getUserGroups($targetUserObject);
 		$gids = [];
 		foreach ($groups as $group) {
+			if (!$this->groupManager->getSubAdmin()->isSubAdminOfGroup($currentLoggedInUser, $group)) {
```

Comment:
```suggestion
			if (!$isAdmin && !$this->groupManager->getSubAdmin()->isSubAdminOfGroup($currentLoggedInUser, $group)) {
```

But not sure we should do that.
It might be important to see that a user is in more/other groups?